packages/twenty-server/src/engine/twenty-orm/decorators/workspace-column-index.decorator.ts

```diff
@@ -0,0 +1,46 @@
+import { generateDeterministicIndexName } from 'src/engine/metadata-modules/index-metadata/utils/generate-deterministic-index-name';
+import { WorkspaceIndexOptions } from 'src/engine/twenty-orm/decorators/workspace-index.decorator';
+import { metadataArgsStorage } from 'src/engine/twenty-orm/storage/metadata-args.storage';
+import { getColumnsForIndex } from 'src/engine/twenty-orm/utils/get-default-columns-for-index.util';
+import { convertClassNameToObjectMetadataName } from 'src/engine/workspace-manager/workspace-sync-metadata/utils/convert-class-to-object-metadata-name.util';
+import { TypedReflect } from 'src/utils/typed-reflect';
+
+export function WorkspaceColumnIndex(
```

Comment:
Why do we need 2 decorators now? You think the code would have been to hard to read with both use cases?